RSignatureVerifier/RSignatureVerifier.swift

```diff
@@ -1,20 +1,31 @@
 /// Signature Verifier public API functions
 @objc public final class RSignatureVerifier: NSObject {
 
+    let realInstance: RealSignatureVerifier
+
+    /// Initializes new instance configured with given parameters
+    /// - Parameter baseURL: URL used to fetch and store public key data
+    /// - Parameter subscriptionKey: key used to authenticate request to `baseURL` endpoint
+    @objc public init(baseURL: URL, subscriptionKey: String) {
+        realInstance = RealSignatureVerifier(fetcher: Fetcher(apiClient: APIClient(),
```

Comment:
Is `realInstance` preferred than `self`?

Reply:
but `self` is not a `RealSignatureVerifier` class/type. I'm not sure what do you mean